packages/dd-trace/src/metrics.js

```diff
@@ -48,11 +48,19 @@ module.exports = {
       nativeMetrics = null
     }
 
-    client = new Client({
+    const clientConfig = {
       host: config.dogstatsd.hostname,
       port: config.dogstatsd.port,
       tags
-    })
+    }
+
+    if (config.url) {
```

Comment:
Why not just pass through everything if it's handled in `request`?

Reply:
It's not. The presence of `metricsProxyUrl` determines whether or not things will go to `request` at all.